helm/cluster-api-provider-azure/values.yaml

```diff
@@ -0,0 +1,19 @@
+name: cluster-api-core
+image:
+  registry: gcr.io
+  name: k8s-staging-cluster-api-azure/cluster-api-azure-controller
```

Comment:
staging?

Reply:
:shrug:

Reply:
This is all just a copypasta from CAPZ repo :)